git-extra/Makefile

```diff
@@ -21,7 +21,7 @@ $(BUILDDIR)/%.res: $(SRCDIR)/%.rc
 	$(WINDRES) --input $< --output $@ --output-format coff
 
 $(BUILDDIR)/WhoUses.exe: $(BUILDDIR)/WhoUses.o $(BUILDDIR)/SystemInfo.o
-	$(CXX) $(CXXFLAGS) -o $@ $(LDFLAGS) $^
+	$(CXX) $(CXXFLAGS) -o $@ $(LDFLAGS) $^ -lmsvcrt-os
```

Comment:
This doesn't seem right.

Reply:
@jeremyd2019 note that it was only necessary with clang's `g++` shim, not with the `gcc` one.